packages/documentation/copy/en/reference/Namespaces and Modules.md

```diff
@@ -70,7 +70,7 @@ This is how the `node.d.ts` file that several of the TypeScript samples use is c
 
 ## Needless Namespacing
 
-If you're converting a program from namespaces to modules, it can be easy to end up with a file that looks like this:
+If you're converting a program from modules to namespaces, it can be easy to end up with a file that looks like this:
```

Comment:
There are `namespaces` in the example result file, so I assume, that it is the result of conversion from `modules` to `namespaces`

Reply:
Looks like my mistake.
Instead of this sentence, example should be fixed